test/functional/api/integrations_controller_test.rb

```diff
@@ -215,6 +215,9 @@ def setup
         "error_auth_missing"=>"ooooh Authentication parameters missing",
         "error_status_no_match"=>504,
         "error_headers_no_match"=>"text/html; charset=us-ascii",
+        "error_status_limits_exceeded"=>499,
+        "error_headers_limits_exceeded"=>"text/html; charset=us-ascii",
+        "error_limits_exceeded"=>"Limit exceeeeeded",
```

Comment:
We're trying not to depend only on functional tests as much as possible. Maybe add something specific to these new attributes in integration tests similar to https://github.com/3scale/porta/blob/06948bbd7118be15b77e561e269fa6b4878fb2bf/test/integration/proxy_example_test.rb#L599?

Reply:
ok, makes sense.

Reply:
added test for lua generator..